vendor/wp-i18n-tools/pomo/streams.php

```diff
@@ -202,8 +202,8 @@ public function POMO_CachedFileReader( $filename ) {
  * Reads the contents of the file in the beginning.
  */
 class POMO_CachedIntFileReader extends POMO_CachedFileReader {
-	public function POMO_CachedIntFileReader( $filename ) {
-		parent::POMO_CachedFileReader( $filename );
+	public function __construct( $filename ) {
```

Comment:
Thanks for pointing that out, Gary. Will submit a pull request for that
later this evening.

2015-12-09 15:05 GMT+01:00 Gary Jones <notifications@github.com>:

> In vendor/wp-i18n-tools/pomo/streams.php
> <https://github.com/cedaro/grunt-wp-i18n/pull/59#discussion_r47093428>:
>
> > @@ -202,8 +202,8 @@ public function POMO_CachedFileReader( $filename ) {
> >   * Reads the contents of the file in the beginning.
> >   */
> >  class POMO_CachedIntFileReader extends POMO_CachedFileReader {
> > -	public function POMO_CachedIntFileReader( $filename ) {
> > -		parent::POMO_CachedFileReader( $filename );
> > +	public function __construct( $filename ) {
>
> This constructor is completely redundant.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cedaro/grunt-wp-i18n/pull/59/files#r47093428>.
>



-- 
Andrea Gandino
Mobile: (+39) 348 43 94 450
Website: http://thehappybit.com | http://andreagandino.com
Twitter: @andre_g